### PR TITLE
Onboarding add background screen

### PR DESCRIPTION
**Pull request by Parveshdhull**
### Summary
Most of the on-boarding screen uses the same background, where we have a slider animation and an overlay above it.
PR adds a background overlay with just a single static image, we can replace that image with a slider animation screen once ready.

status: ready

### CODE REVIEW

**Comment by flexsurfer** [comment]
related PRs:
https://github.com/status-im/status-mobile/pull/15127
https://github.com/status-im/status-mobile/pull/15110

**Comment by flexsurfer** [comment]
cc @J-Son89

**Comment by Parveshdhull** [comment]
As this background is used in most of the onboarding screens, the purpose of this PR is to implement this as a separate screen and avoid duplicate work.

This PR still needs a few improvements, if the implementation for this background screen is already ready please isolate that as a standalone component/screen and create a separate PR. So that it can be used on other screens.

And please feel free to close it and let me know if I should continue working on it.

cc: @J-Son89 @ulisesmac @briansztamfater

**Comment by ulisesmac** [comment]
> As this background is used in most of the onboarding screens, the purpose of this PR is to implement this as a separate screen and avoid duplicate work.
> 
> This PR still needs a few improvements, if the implementation for this background screen is already ready please isolate that as a standalone component/screen and create a separate PR. So that it can be used on other screens.
> 
> And please feel free to close it and let me know if I should continue working on it.
> 
> cc: @J-Son89 @ulisesmac @briansztamfater 

Your implementation is looking solid, and it's a good idea to isolate the component.

I created the component too for a closed PR (it's not merged, I closed it until a blocker is solved), my implementation consists of just an image using blur-radius and an overlay (as you can see, it's pretty basic).

IMO you can continue working on this, just make sure it will work well with what @J-Son89 did on #15127

File: src/status_im2/contexts/onboarding/common/background.cljs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  [rn/view
+   {:style style/background-container}
+   ;; Todo - add carousel component as background once ready
+   ;; https://github.com/status-im/status-mobile/issues/15012
+   [rn/image
+    {:blur-radius (if dark-overlay? 13 0)
+     :style       {:flex 1}
+     :source      (resources/get-image :intro-1)}]
+   [linear-gradient/linear-gradient
+    {:colors [(if dark-overlay? (colors/custom-color :yin 50) "#000716")
```
Comment (Parveshdhull) [inline src/status_im2/contexts/onboarding/common/background.cljs:19]: https://github.com/status-im/status-mobile/pull/15127#discussion_r1125162680

**Review by J-Son89 — APPROVED** [review]

**Pull request merged**